### ISSUE TITLE
Access VPN network blocked by rules rules

### ISSUE THREAD

**Issue by bilby91** [aws/amazon-vpc-cni-k8s#133]
Hello,

We are migrating our infrastructure to Kubernetes and we are exploring the EKS solution from AWS. I'm working on getting the Kubernetes cluster to communicate with our On Premise services via OpenVPN. OpenVPN client is running on all worker nodes correctly. The problem I'm having is that Kubelet is adding iptables rules that are blocking the traffic from the node to the On Premise servers. Is there any way to allow this outbound traffic ? 

This might not be related to CNI itself so please let me know and I'll close the issue.

Thanks before hand!

**Comment by liwenwu-amazon** [comment]
@bilby91 With release 1.1, we have added a feature which allows users to disable on-node SNAT.  https://aws.amazon.com/blogs/opensource/vpc-cni-plugin-v1-1-available/ gives an overview on how this feature works.

**Comment by bilby91** [comment]
@liwenwu-amazon Thanks for the feedback. I will definitely try this. 

I was able to get my desired networking on a non eks VPC. I basically added a NAT node that forwards traffic to my VPN over a tunnel in the NAT node. Then I had to add a route to each node in the VPC to route traffic for VPN network over the NAT instance. 

After reviewing the blog post I'm not sure if I need to disable the SNAT on my nodes. 

Adding a new route on each EKS node to route traffic over my NAT node wouldn't be enough ? If that is not enough, disabling the SNAT on the nodes wouldn't break my nodes internet connection ?

Thanks before hand!
